projects/storefrontapp-e2e-cypress/cypress/integration/regression/my-account/consent-management.e2e-spec.ts

```diff
@@ -0,0 +1,42 @@
+import { viewportContext } from '../../../helpers/viewport-context';
+import {
+  consentManagementTest,
+  verifyAsAnonymous,
+} from '../../../helpers/consent-management';
+import * as login from '../../../helpers/login';
+
+viewportContext(['mobile', 'desktop'], () => {
+  describe('Consent Management Page', () => {
+    before(() => {
+      cy.window().then((win) => win.sessionStorage.clear());
+    });
+
+    describe('consent management test for anonymous user', () => {
+      verifyAsAnonymous();
+    });
+
+    describe('consent management test for logged in user', () => {
+      before(() => {
+        cy.requireLoggedIn();
+        cy.reload();
```

Comment:
Is this reload required?

Reply:
I migrated over the tests that were in the mobile folder, I didn't modify any of the tests since it was out of scope for the ticket and I had/have other tasks to work on.

I think all the tests that were migrated can be improved. I will make a ticket for that if you agree we can do it outside of this PR.